lib/plug/conn.ex

```diff
@@ -984,7 +984,13 @@ defmodule Plug.Conn do
         plug_status: 414
     end
 
-    query_params = Plug.Conn.Query.decode(query_string, %{}, Plug.Conn.InvalidQueryError)
+    query_params =
+      Plug.Conn.Query.decode(
+        query_string,
+        %{},
+        Plug.Conn.InvalidQueryError,
+        Keyword.get(opts, :validate_utf8_query, true)
```

Comment:
Looks good to me. I would just call the option `:validate_utf8`. Can you please adjust it accordingly? Thanks!

Reply:
I've named it this way so that it doesn't collide with the `validate_utf8` flag of the multipart parser. If I understand well, all global options of the `Plug.Parsers` will be applied to all parsers. Setting this flag  would set the `validate_utf8` flag on the multipart parser too. I just want to make sure that its what we want.

Reply:
Yeah, i generally think someone would want to disable it altogether. :+1:

Reply:
It make sense.

Reply:
I have made the change, can you review it? Maybe I shouldn't have rebased my code to highlight the changes that I have made.